charts/dependabot-gitlab/values.yaml

```diff
@@ -113,6 +113,22 @@ worker:
     # -- Startup probe timeout
     timeoutSeconds: 3
 
+  serviceMonitor:
```

Comment:
The way `helm-docs` populates values and their description in Readme, is by reading comments prefixed with `# --`, please add such a comment to every value like the rest of the values in this file